### PR TITLE
peek generators and setup 

### PR DESCRIPTION
**Pull request by kubalasecki**
work in progress

### CODE REVIEW

File: lib/netguru/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Netguru
-  VERSION = "0.0.7.4"
+  VERSION = "0.0.7.5"
```
Comment (lubieniebieski) [inline lib/netguru/version.rb]: it could be even `0.0.8`!

Comment (kubalasecki) [inline lib/netguru/version.rb]: so it's a big improvment ? wow :D

File: netguru.gemspec [hunk 2]
```diff
@@ -20,6 +20,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rspec"
   s.add_development_dependency "webmock"
   s.add_development_dependency "pry"
+  s.add_development_dependency "peek"
```
Comment (lubieniebieski) [inline netguru.gemspec]: should be as regular dependency, not development one (same as the others)

**Comment by madsheep** [comment]
what's hte status here?

**Comment by madsheep** [comment]
:sheep:

**Comment by lukkry** [comment]
:blue_heart:

**Pull request closed without merging**